core/services/bulletprooftxmanager/eth_broadcaster.go

```diff
@@ -442,7 +442,8 @@ func saveFatallyErroredTransaction(store *store.Store, etx *models.EthTx) error
 		if err := tx.Exec(`DELETE FROM eth_tx_attempts WHERE eth_tx_id = ?`, etx.ID).Error; err != nil {
 			return errors.Wrapf(err, "saveFatallyErroredTransaction failed to delete eth_tx_attempt with eth_tx.ID %v", etx.ID)
 		}
-		return errors.Wrap(tx.Save(etx).Error, "saveFatallyErroredTransaction failed to save eth_tx")
+		// Omit the EthTxAttempts (we just deleted them) or gorm v2 will try to save them.
```

Comment:
Can we please just disable autosave associations instead?

Reply:
I'm not aware of a way aside from Omit - https://gorm.io/docs/associations.html#comment-5072455528

Reply:
fwiw it won't _update_ the association data, but auto creates are on by default and jinzhu says to use Omit

Reply:
Horrible thing